### ISSUE TITLE
wrong syntax detection for castsequence macro

### ISSUE THREAD

**Issue by LootFever** [NumyAddon/MacroToolkit#63]
The syntax for a cast sequence is as follows:
`/castsequence [options] reset=condition1/... action1, action2, ...`

wich means a macro like this
```
#showtooltip
/castsequence [known:6343,known:260643] reset=target Thunder Clap, Skullsplitter;[known:260643] reset=target Rend,Skullsplitter
```
should be detected correctly. The macro itself is correct as the game executes it like it should.

But Macro Toolkit does not detect it correct:
![image](https://github.com/user-attachments/assets/9c6b7023-0ced-4566-b46d-b25ab19a217d)

**Comment by Numynum** [comment]
Is it only showing those `known spell mismatch` warnings?
because those are warnings, not errors (though I know the UI doesn't really clarify this)

But at the end of the day, those warnings should be changed to either work properly for cast sequences, or not show at all

**Comment by LootFever** [comment]
> Is it only showing those `known spell mismatch` warnings? because those are warnings, not errors (though I know the UI doesn't really clarify this)
> 
> But at the end of the day, those warnings should be changed to either work properly for cast sequences, or not show at all

Yeah I guess those are just warnings, but then I never called them errors 😄 
The addon doesn't hinder editing or anything.
